src/Migrate.php

```diff
@@ -102,6 +112,23 @@ private function restoreDestinationProject(array $restoreCredentials): void
         $this->logger->info('Current project restored');
     }
 
+    private function migrateDataOfTablesDirectly(): void
+    {
+        $this->logger->info('Migrate data of tables directly.');
+
+        $this->destJobRunner->runJob(
+            Config::DATA_OF_TABLES_MIGRATE_COMPONENT,
+            [
+                'parameters' => [
+                    'sourceKbcUrl' => $this->sourceProjectUrl,
+                    '#sourceKbcToken' => $this->sourceProjectToken,
+                ],
+            ]
```

Comment:
Netreba tam niekde esten `runId`? Aby sme vedeli ako job vznikol? Niekde sa to tak robilo, nie som si isty ....

Reply:
Mozno sa spytat v PS?

Reply:
@webrouse no ten se nastavuje tady:
https://github.com/keboola/app-project-migrate/blob/master/src/JobRunner/SyrupJobRunner.php#L35

Reply:
jj, ale nemal by sa pouzit rovnaky? tj. aby tie jobs, mali rovnake `RUN_ID`?
Asi nic, ... mal to tak asi iba stary gooddata wrier. Ok. :+1: